src/main/java/me/clip/placeholderapi/util/Futures.java

```diff
@@ -40,11 +40,8 @@ public static <T> void onMainThread(@NotNull final Plugin plugin,
       @NotNull final CompletableFuture<T> future,
       @NotNull final BiConsumer<T, Throwable> consumer) {
     future.whenComplete((value, exception) -> {
-      if (Bukkit.isPrimaryThread()) {
-        consumer.accept(value, exception);
-      } else {
-        Bukkit.getScheduler().runTask(plugin, () -> consumer.accept(value, exception));
-      }
+      if (Bukkit.isPrimaryThread()) consumer.accept(value, exception);
+      else Bukkit.getScheduler().runTask(plugin, () -> consumer.accept(value, exception));
```

Comment:
Why remove the brackets and format it like this. It just makes it more difficult to read.